.github/ffm-audit/audit.py

```
@@ -83,11 +84,82 @@ def main():
+        layouts = sorted(x for x in sskip if isinstance(x, str))
+        fields = sorted('%s.%s' % x for x in sskip if isinstance(x, tuple))
+        if layouts:
+            print('  no SDK declaration: ' + ', '.join(layouts))
+        if fields:
+            print('  fields absent from the SDK struct: ' + ', '.join(fields))
+    # Anything either pass reported that is neither a recognized finding nor a documented skip means
+    # the harness itself is broken on this platform, and must not be mistaken for a clean run. Both
+    # passes need this: a structs.cpp that fails to compile for an unexpected reason produces no
+    # cmp<> findings at all, which would otherwise read as a clean struct audit.
+    def unexpected(text, recognized):
+        return [l for l in text.splitlines()
```

Ruff (0.16.1)

[error] Ambiguous variable name: `l`

(E741)